pkg/manifests/config.go

```diff
@@ -92,6 +92,36 @@ type PrometheusK8sConfig struct {
 	VolumeClaimTemplate *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate"`
 	RemoteWrite         []monv1.RemoteWriteSpec              `json:"remoteWrite"`
 	TelemetryMatches    []string                             `json:"-"`
+	AlertmanagerConfigs []AdditionalAlertmanagerConfig       `json:"additionalAlertManagerConfigs"`
+}
+
+type AdditionalAlertmanagerConfig struct {
+	// The URL scheme to use when talking to Alertmanagers.
+	Scheme string `json:"scheme,omitempty"`
+	// Path prefix to add in front of the push endpoint path.
+	PathPrefix string `json:"pathPrefix,omitempty"`
+	// The timeout used when sending alerts.
+	Timeout *string `json:"timeout,omitempty"`
+	// The api version of Alertmanager.
+	APIVersion string `json:"apiVersion"`
+	// TLS Config to use for alertmanager connection.
+	TLSConfig TLSConfig `json:"tlsConfig,omitempty"`
+	// Bearer token to use when authenticating to Alertmanager.
+	BearerToken string `json:"bearerToken,omitempty"`
```

Comment:
It would be better to have a reference to a secret's key rather than the token stored as plain-text in the configmap.

Reply:
Can we support both? I mean that I can have another PR to support a secret key as `BearerTokenFile`.

Reply:
My concern is that the token value is exposed in clear text in the CMO configmap. I'm fine with the final Prometheus configuration using `bearer_token`.

Reply:
I have changed the bearerToken to be a secret key selector. and use `authorization` in Promtheus configuration since `bearer_token` and `bearer_token_file` marked as deprecated.